socorro/cron/config.py

```diff
@@ -0,0 +1,17 @@
+"""Default configuration values used by the Crontabber app."""
+from configman.dotdict import DotDict
+
+
+# Ignore non-config module members like DotDict
+always_ignore_mismatches = True
+
+crontabber = DotDict({
+    'database_class': 'socorro.external.postgresql.connection_context.ConnectionContext',
+})
+
+
+resource = DotDict({
+    'postgresql': {
+        'database_class': 'socorro.external.postgresql.connection_context.ConnectionContext',
+    }
+})
```

Comment:
This seems confusing to me because configuration keys end up defined using a mix of module-level variable names and DotDict keys.

If we do this instead:

```
klass_config = 'socorro.cron.config.CONFIG'
```

in the class and then have this file be:

```
CONFIG = DotDict({
    'crontabber': {
        'database_class': 'socorro.external.postgresql.connection_context.ConnectionContext',
    },
    'resource': {
        'postgresql': {
            'database_class': 'socorro.external.postgresql.connection_context.ConnectionContext',
        }
    }
})
```

Then all the keys for a configuration are keys rather than a mix of module-level names and DotDict keys.

One nice thing about that is that we could have multiple configurations defined in the same Python file, though we don't need that now.